core-ui/src/shared/components/PersistentVolumesList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ComponentForList } from 'shared/getComponents';
+
+export function PersistentVolumesList(storageclass) {
```

Comment:
I think it would also make sense to add a status column here.

Reply:
We have to discuss it because I have one question. 😄  
(Comment just to not forget)

Reply:
As we talked, I have to check it after merging Jakub's PR.